dev-tools/scripts/smokeTestRelease.py

```diff
@@ -861,88 +756,40 @@ def testSolrExample(unpackPath, javaPath, isSrc):
   else:
     os.chdir(unpackPath)
     
-# check for broken links
-def checkBrokenLinks(path):
-  # also validate html/check for broken links
-  if checkJavadocLinks.checkAll(path):
-    raise RuntimeError('broken javadocs links found!')
-
-def testDemo(run_java, isSrc, version, jdk):
```

Comment:
ah! here's the `testDemo` method being removed which the `pylint` was flagging up as undefined at line 630 above.